virttest/qemu_monitor.py

```diff
@@ -2222,9 +2222,9 @@ def send_args_cmd(self, cmdlines, timeout=CMD_TIMEOUT, convert=True):
                                 value = int(value)
                         elif re.match("^[0-9]+\.[0-9]*$", value):
                             value = float(value)
-                        elif re.findall("true", value, re.I):
+                        elif value.lower() in ["on", "yes", "y", "true"]:
```

Comment:
Well, the question is: is it true for all the usages that any string in the list has to be converted to bool values? for example, what if I want `id=y`, is it acceptable?

Reply:
Nice catch. `y` and `n` have no clear semantics in other scenarios, I think can move it from the list, only use `yes` directly.